src/nodejs/neo-rpc/v2.9.0/client/module/getNodesBy.js

```diff
@@ -212,7 +227,19 @@ exports.connection = (options) => {
   pingOptions.order = 'asc'
 
   return new Promise((resolve, reject) => {
-    this.ping(pingOptions).then(nodes => {
+    if (parseInt(options.ping)) {
+      this.ping(pingOptions).then(nodes => {
+        connectionCallback(nodes)
+      })
+      .catch (error => {
+        print(__filename + ': getNodesBy.connections()/getNodesBy.ping().error: ' + error)
+      })
+    }
+    else {
+        connectionCallback(options.nodes)
+    }
+
+    function connectionCallback(nodes) {
```

Comment:
can't you reuse the `tallestCallback`?

Reply:
Nah, these will eventually be reused from outside of their API expression too. They need to be unique for extensibility, for now.